src/App.js

```diff
@@ -1,15 +1,134 @@
-import React from 'react';
+import React, { Component } from 'react';
+import Button from './components/Button/Button';
+import TodoItemSort from './components/TodoItemSort/TodoItemSort';
+import TodoList from './components/TodoList/TodoList';
+
 import './App.css';
 
-import todos from './api/todos';
-import users from './api/users';
+const TODOS_URL = 'https://jsonplaceholder.typicode.com/todos';
+const USERS_URL = 'https://jsonplaceholder.typicode.com/users';
+
+class App extends Component {
+  state = {
+    todos: [],
+    users: [],
+    sort: 'default',
+    isLoading: false,
+  };
+
+  getTodosWithUsers = (todosList, usersList) => todosList.map(todo => ({
+    ...todo,
+    user: usersList.find(user => user.id === todo.userId),
+  }))
+
+  handleShow = () => {
+    this.setState({
+      isLoading: true,
+    });
+
+    Promise.all([
+      fetch(TODOS_URL),
+      fetch(USERS_URL),
+    ])
+      .then(([resTodos, resUsers]) => Promise.all(
+        [resTodos.json(), resUsers.json()]
+      ))
+      .then(([dataTodos, dataUsers]) => this.setState({
+        todos: dataTodos,
+        users: dataUsers,
+        isLoading: false,
+      }));
+  };
+
+  sort = (items, sort) => {
+    switch (sort) {
+      case 'default':
```

Comment:
what is the difference between this case and default case?

Reply:
fixed